modules/nw-routeadvertisements-example.adoc

```diff
@@ -0,0 +1,254 @@
+// Module included in the following assemblies:
+//
+// * networking/route_advertisements/about-bgp-routing.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-bgp-examples_{context}"]
+= Example advertising pod IP addresses with BGP
+
+The following examples describe several configurations for advertising pod IP addresses with Border Gateway Protocol (BGP). The external network border router has the `172.18.0.5` IP address.
+
+Each example relies upon the following `FRRConfiguration` object:
+
+.`FRRConfiguration` CR
+[source,yaml]
+----
+apiVersion: frrk8s.metallb.io/v1beta1
+kind: FRRConfiguration
+metadata:
+  name: receive-all
+  namespace: openshift-frr-k8s
+  labels:
+    routeAdvertisements: receive-all
+spec:
+  bgp:
+    routers:
+    - asn: 64512
+      neighbors:
+      - address: 172.18.0.5
+        asn: 64512
+        disableMP: true
+        toReceive:
+          allowed:
+            mode: filtered
+----
+
+The OVN-Kubernetes controller checks that the `RouteAdvertisements` CR selected nodes are a subset of the nodes selected by the `RouteAdvertisements` CR selected FRRConfigurations.
+
+[id="advertising-pod-ips-from-a-user-defined-network-over-bgp_{context}"]
+== Advertising pod IPs from a cluster user-defined network over BGP
+
+In this scenario, the blue cluster user-defined network (CUDN) is exposed to the external network so that the attached pods have IP addresses that are reachable from the external network. The cluster user-defined networks are each attached to a different VRF device as illustrated in the following diagram:
+
+image::524-openshift-bgp-ovn-k8s-no-vpn-0325.png[Advertising pod IPs from a cluster user-defined network over BGP]
+
+Red CUDN::
+- A VRF named `red`
```

Comment:
There is one detail that is potentially worth mentioning. 

The VRF for a CUDN is named after the CUDN name as long as has less than 16 characters length. Otherwise the VRF name randomly defined. 

This will be improved in the future but practically means a user should be using short names for CUDNs planned to be advertised.